code/datums/wires/_wires.dm

```diff
@@ -33,6 +33,9 @@
 	var/list/assemblies = list() // List of attached assemblies.
 	var/randomize = 0 // If every instance of these wires should be random.
 					  // Prevents wires from showing up in station blueprints
+	var/req_knowledge = INFINITY //wiring skill level on which the functions are revealed.
+	var/req_skill = JOB_SKILL_BASIC //used in user's cutting/pulsing/mending speed calculations.
```

Comment:
this should be untrained by default, either that or airlocks need to be untrained.